src/test/java/seedu/address/model/person/TissuetypeTest.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class TissuetypeTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Tissuetype(null));
+    }
+
+    @Test
+    public void constructor_invalidTissuetype_throwsIllegalArgumentException() {
+        String invalidTissuetype = "";
+        assertThrows(IllegalArgumentException.class, () -> new Tissuetype(invalidTissuetype));
+    }
+
+    @Test
+    public void isValidTissuetype() {
```

Comment:
Just wondering, if I have an input string "1, 2, 3, 4, 5, 6", would it count?

Reply:
yes, line 38 tested it

Reply:
I meant if the input string will accept the inputs separated by a comma and a space. Line 38 tests if the inputs are separated by commas.